src/modules/StudyManager/components/List/List/components/Table/style.scss

```diff
@@ -71,6 +65,9 @@
 
 	&cell-name {
 		display: flex;
+		&__title {
+			@include text-ellipsis();
+		}
 	}
 
-}
```

Comment:
Let's avoid changing spacing EOF markers , unless we have a written policy on these. If we do, please direct me to it.